openshift_images/cnf-building-and-deploying-a-dpdk-payload.adoc

```diff
@@ -131,25 +153,149 @@ spec:
 
 <3> The `source` type contains the git repository and a context directory within the repository.
 
-<4> The `strategy` type contains a DPDK base image.
+<4> The repository uri that contains the application and both the `build.sh` and the `run.sh` files.
+
+<5> The `strategy` type contains a DPDK base image.
 
 It is the `source` type and `strategy` type that build the image.
 
 A complete guide to using BuildConfigs is available in
 xref:../builds/understanding-buildconfigs.adoc#understanding-buildconfigs[Understanding
 build configurations].
 
-After the base DPDK image build is ready, create a new directory with a
-`kustomization.yaml` file and a `build-config.yaml` patch.
+After the base DPDK image build is ready, you should configure the environment to be able to run the DPDK workload on it.
 
-This is an example of `kustomization.yaml`:
+.Deployment procedure
 
+You should create a performance profile
+(See [https://docs.openshift.com/container-platform-ocp/4.4/scalability_and_performance/cnf-performance-addon-operator-for-low-latency-nodes.html#cnf-creating-the-performance-profile-object_cnf-master]) To allocate Hugepages and isolated CPUs.
+
+Then you should create both SR-IOV network policy and SR-IOV network attachment
+(See https://docs.openshift.com/container-platform-ocp/4.4/networking/hardware_networks/using-dpdk-and-rdma.html[using-dpdk-and-rdma]) based on your network card type.
```

Comment:
xref:..networking/hardware_networks/using-dpdk-and-rdma.adoc#using-dpdk-and-rdma[Using Virtual Functions (VFs) with DPDK and RDMA modes]

Reply:
Updated xref